css/src/components/admin-list.css

```diff
@@ -1,45 +1,76 @@
 /**
- * Admin lists.
+ * @file
+ * Admin list and admin item modules.
  */
-ul.admin-list {
-  margin: 0;
+
+/**
+ * Admin list (wrapper of admin items).
+ */
+.admin-list {
+  margin-top: 1em;
+  margin-bottom: 2em;
   padding: 0;
+  border: calc(1em / 16) solid var(--color-lightgray);
```

Comment:
It doesn't seem like this is needed at all. We should be able to rely on the border set for the `.admin-item` element.